packages/runtime/container-runtime/src/test/attributor/runtimeAttributor.spec.ts

```diff
@@ -5,43 +5,113 @@
 
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 
-import { strict as assert } from "node:assert";
+import { strict as assert } from "assert";
 
+import {
+	type IProvideRuntimeAttributor,
+	enableOnNewFileKey,
+	type IAttributor,
+	AttributorSerializer,
+	chain,
+	deltaEncoder,
+	makeLZ4Encoder,
+	Attributor,
+} from "@fluidframework/attributor/internal";
 import {
 	AttachState,
 	type ICriticalContainerError,
 } from "@fluidframework/container-definitions";
-import { type IContainerContext } from "@fluidframework/container-definitions/internal";
+import {
+	type IAudience,
+	type IContainerContext,
+} from "@fluidframework/container-definitions/internal";
 import { type ConfigTypes, type FluidObject } from "@fluidframework/core-interfaces";
-import { SummaryType } from "@fluidframework/driver-definitions";
 import {
 	type IDocumentStorageService,
 	type ISnapshotTree,
 	type ISequencedDocumentMessage,
+	SummaryType,
+	type IQuorumClients,
+	type ISequencedClient,
+	type IClient,
 } from "@fluidframework/driver-definitions/internal";
+import { FlushMode } from "@fluidframework/runtime-definitions/internal";
 import {
 	MockLogger,
 	sessionStorageConfigProvider,
 } from "@fluidframework/telemetry-utils/internal";
-import { MockDeltaManager } from "@fluidframework/test-runtime-utils/internal";
-
-import { Attributor } from "../attributor.js";
-import { AttributorSerializer, chain, deltaEncoder } from "../encoders.js";
-import { makeLZ4Encoder } from "../lz4Encoder.js";
 import {
-	type IProvideRuntimeAttributor,
-	createRuntimeAttributor,
-	enableOnNewFileKey,
-	mixinAttributor,
-} from "../mixinAttributor.js";
+	MockDeltaManager,
+	MockQuorumClients,
+} from "@fluidframework/test-runtime-utils/internal";
 
-import { makeMockAudience, makeMockQuorum } from "./utils.js";
+import { ContainerRuntime } from "../../index.js";
 
 type Mutable<T> = {
 	-readonly [P in keyof T]: T[P];
 };
 
-describe("mixinAttributor", () => {
+/**
+ * Creates a mock {@link @fluidframework/protocol-definitions#IQuorumClients} for testing.
+ */
+export function makeMockQuorum(clientIds: string[]): IQuorumClients {
```

Comment:
Can we not have it here please? We should place mocks in packages where it allows their reuse across different features that need mocks. Take a look how we do mocks for Audience and other things.